roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -205,6 +205,7 @@ kiali_defaults:
 
   kiali_feature_flags:
     istio_injection_action: true
+    istio_upgrade_action: false
```

Comment:
Even though the other two settings are not defined by default (I am assuming this to be true), you still should put them in here for documentation purposes, for developers to see.

So up around lines [154-160](https://github.com/kiali/kiali-operator/blob/v1.36.0/roles/default/kiali-deploy/defaults/main.yml#L154-L160), add these two settings, but comment them out (just like we do for things like [label_selector as seen here](https://github.com/kiali/kiali-operator/blob/v1.36.0/roles/default/kiali-deploy/defaults/main.yml#L28)):

```
      envoy_admin_local_port: 15000
      #istio_current_revision:
      istio_identity_domain: "svc.cluster.local"
      istio_injection_annotation: "sidecar.istio.io/inject"
      istio_sidecar_annotation: "sidecar.istio.io/status"
      istio_sidecar_injector_config_map_name: "istio-sidecar-injector"
      #istio_upgrade_revision:
      istiod_deployment_name: "istiod"
      url_service_version: ""
```